### ISSUE TITLE
glob doesn't work for same attribute value nodes

### ISSUE THREAD

**Issue by Odelya** [c0fec0de/anytree#3]
Since __get_nodemap returns OrderedDict, it requires that the search by part will be unique, and hence doesn't allow searching for more than one node with the same value.

E.g.

```
d = Node("parent")
child_1 = Node("child_1", parent=d, value="5")
child_2 = Node("child_2", parent=d, value="5")
r = Resolver("value")
nodes = r.glob(d, "5")
```

will return only the last found node.

**Comment by c0fec0de** [comment]
Thanks for contributing. The Resolver().get() method has the same issue. It shall return the first item. Currently it is the last.

**Comment by c0fec0de** [comment]
Fixed in revision 2.1.1. The pypi package is out now.
